packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -0,0 +1,153 @@
+                        </div>
+                        <div className='flex flex-col'>
+                            <div className='flex items-center flex-row gap-300'>
+                                <Text size='md'>
+                                    {platform === mt5Platform
+                                        ? MarketTypeDetails[(marketType ?? 'all') as keyof typeof MarketTypeDetails]
+                                              .title
+                                        : PlatformDetails[platform as keyof typeof PlatformDetails].title}{' '}
+                                    {!activeAccount?.is_virtual && details?.landing_company_short?.toUpperCase()}
+                                </Text>
+                            </div>
+                            <Text className='text-system-light-less-prominent-text' size='sm'>
+                                {loginId}
+                            </Text>
+                        </div>
+                    </div>
+                    <div className='flex flex-col items-end'>
+                        <Text bold>{details?.display_balance}</Text>
+                    </div>
+                </div>
+                <div className='flex flex-col gap-100'>
+                    {platform === mt5Platform && (
+                        <Fragment>
+                            <MT5TradeDetailsItem label='Broker' value='Deriv Holdings (Guernsey) Ltd' />
+                            <MT5TradeDetailsItem
+                                label='Server'
+                                value={(details as THooks.MT5AccountsList)?.server_info?.environment ?? 'Deriv-Server'}
+                            />
+                            <MT5TradeDetailsItem label='Login ID' value={loginId ?? '12345678'} />
+                            <MT5TradeDetailsItem label='Password' value='********' variant='password' />
+                        </Fragment>
+                    )}
+                    {platform === dxtradePlatform && (
+                        <Fragment>
+                            <MT5TradeDetailsItem
+                                label='Username'
+                                value={(details as THooks.DxtradeAccountsList)?.login ?? '12345678'}
+                            />
+                            <MT5TradeDetailsItem label='Password' value='********' variant='password' />
+                        </Fragment>
+                    )}
+                    {platform === ctraderPlatform && (
+                        <MT5TradeDetailsItem
+                            value=' Use your Deriv account email and password to login into the cTrader platform.'
+                            variant='info'
+                        />
+                    )}
+                </div>
+                <div className='flex gap-400'>
+                    <ImportantIcon />
+                    <Text size='2xs'>{serviceMaintenanceMessages[(platform || mt5Platform) as TPlatforms.All]}</Text>
```

Comment:
@sergei-deriv do we have this text size in quill? 2xs?

Reply:
@mahdiyeh-deriv my bad, forgot to change size from previous `WalletText`